Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM amazoncorretto:11-alpine-jdk
+
+RUN mkdir /data
+WORKDIR /data
+COPY . /data
+
+RUN ./gradlew build
+
+EXPOSE 8080
```

Comment:
That Dockerfile definitely lacks some environment variables definition. Your application should be able to get a configuration for database at least...

Reply:
Configuration for database is located at ```docker-compose.yml```